assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.scss

```diff
@@ -40,5 +69,5 @@
   bottom: 0;
   left: 0;
   right: 0;
-  background:rgba(0,0,0,0.6);
+  background:transparent;
```

Comment:
We're not darkening the background?

Reply:
Nope. @Amohkhan are you still happy with that?